std/array.d

```diff
@@ -3173,7 +3174,51 @@ if (isDynamicArray!A)
         }
     }
 
+    /**
+     * Gives a string in the form of `Appender!(A)(data)`.
+     *
+     * Params:
+     *     w = A `char` accepting
+     *     $(REF_ALTTEXT output range, isOutputRange, std, range, primitives).
+     *     fmt = A $(REF FormatSpec, std, format) which controls how the array
+     *     is formatted.
+     * Returns:
+     *     A `string` if `writer` is not set; `void` otherwise.
+     */
+    string toString()() const
+    {
+        // Even though the Appender might represent a string,
+        // make a copy of the data anyway because we don't want
+        // manipulations of the toString result to change the
+        // data of the Appender
```

Comment:
If the element type is immutable how could manipulations of the toString result change the data of the Appender?

Reply:
hmmm, you're right, I was thinking more of the `char[]` case. In any case, the code remains the same regardless to keep it dry and to use the speedup of Appender.

I'll just remove the comment.